pomm-project/pomm-bundle/2.0/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "alias": "pomm",
```

Comment:
Ok, the bot fails fine with the right key:
![2017-05-02-152517_1676x993_scrot](https://cloud.githubusercontent.com/assets/113045/25619613/ccdaaa68-2f4b-11e7-8c10-cd29042e2f38.png)